### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by andreicioban**
### What changed?

* fixes a typo in DevelopedBy component
* Added back the required onClick handler for subscribe button

### CODE REVIEW

File: src/components/subscribe/subscribe-form.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ import { Button } from "../button/button";
 
 export const SubscribeForm = ({ compact }) => {
   const SubscribeButton = () => (
-    <Button inputType="submit">{Wording.BUTTON}</Button>
+    <Button inputType="submit" onClick={() => void 0}>
```
Comment (moonflare) [inline src/components/subscribe/subscribe-form.js]: Oh. My bad. I asked for this. It was actually another component that had this removed and I just messed them in my mind.

File: src/components/developed-by/developed-by.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export const DevelopedBy = ({ showSecondLine }) => {
     <>
       <div className="container developed-by">
         <div>
-          <div classNamne="caption">Un proiect în parteneriat cu </div>
+          <div className="caption">Un proiect în parteneriat cu </div>
```
Comment (moonflare) [inline src/components/developed-by/developed-by.js]: This is also my bad, but it was fixed in another PR, must have been re-added on merge conflict

File: src/components/subscribe/subscribe-form.js (hunk 1 above)
Comment (moonflare) [reply src/components/subscribe/subscribe-form.js]: I guess in the future we can make this callback optional in the `Button` component so we don't have to "fix" it this way in different places

Comment (andreicioban) [reply src/components/subscribe/subscribe-form.js]: No problem. It happens when you do lots of things at once.

**Pull request merged**